docs/source/bft_configuration.md

```diff
@@ -0,0 +1,228 @@
+* `RequestBatchMaxInterval`: The maximal time interval a request batch can wait before it is proposed. A request batch is accumulating requests until `RequestBatchMaxInterval` had elapsed from the time the batch was first created (i.e. the time the first request was added to it), or until it is of count `RequestBatchMaxCount`, or it reaches `RequestBatchMaxBytes`, whichever occurs first.
+* `IncomingMessageBufferSize`: The size of the buffer holding incoming messages before they are processed (maximal number of messages).
+* `RequestPoolSize` : The number of pending requests retained by the node. The `RequestPoolSize` is recommended to be at least double (x2) the `RequestBatchMaxCount`. This cannot be changed dynamically and the node must be restarted to pick up the change.
+* `RequestForwardTimeout`: Is started from the moment a request is submitted, and defines the interval after which a request is forwarded to the leader.
+* `RequestComplainTimeout`: Is started when `RequestForwardTimeout` expires, and defines the interval after which the node complains about the view leader.
+* `RequestAutoRemoveTimeout`: Is started when `RequestComplainTimeout` expires, and defines the interval after which a request is removed (dropped) from the request pool.
+* `ViewChangeResendInterval`: Defines the interval after which the ViewChange message is resent.
+* `ViewChangeTimeout`: Is started when a node first receives a quorum of `ViewChange` messages, and defines the interval after which the node will try to initiate a view change with a higher view number.
+* `LeaderHeartbeatTimeout`: Is the interval after which, if nodes do not receive a "sign of life" from the leader, they complain about the current leader and try to initiate a view change. A sign of life is either a heartbeat or a message from the leader.
+* `LeaderHeartbeatCount`: Is the number of heartbeats per `LeaderHeartbeatTimeout` that the leader should emit. The heartbeat-interval is equal to: `LeaderHeartbeatTimeout/LeaderHeartbeatCount`.
+* `CollectTimeout`: Is the interval after which the node stops listening to `StateTransferResponse` messages, stops collecting information about view metadata from remote nodes.
+
+
+## Block validation policy 
+The block validation policy in a Fabric channel defaults to a policy that requires a signature from any orderer:
+```
+        # BlockValidation specifies what signatures must be included in the block
+        # from the orderer for the peer to validate it.
+        BlockValidation:
+            Type: ImplicitMeta
+            Rule: "ANY Writers"
+```
+
+In BFT, the [configtxgen](./commands/configtxgen.md) tool encodes a policy that is suitable for BFT.
+However, when adding or removing ordering service nodes from the channel,
+the policy should be adjusted accordingly. 
```

Comment:
This seems important, but not enough information is provided.
What policy is "suitable for BFT"?
What adjustment should be made when adding ordering service nodes?
A conceptual description could be provided, and then you could point to the sample as an example - 
fabric-samples/test-network/scripts/add_new_orderer_to_config.py

Reply:
added